### PR TITLE
Add mangatx plugin

### PR DESCRIPTION
**Pull request by robzimpulse**
This should work for all manga on [Mangatx](https://mangatx.com/) and other manga hosting that using word press (ex: [Mangaclash](https://mangaclash.com/)).

Let me know what you think!

Thank you for making such an awesome program :)

### CODE REVIEW

File: plugins/mangatx/info.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+	"id": "Mangatx",
+	"title": "Mangatx",
+	"author": "Robzimpulse - kugelfang.killaruna@gmail.com",
+	"version": "v1.0",
+	"placeholder": "Mangatx Gallery URL",
```
Comment (hkalexling) [inline plugins/mangatx/info.json]: Perhaps it would be clearer to use "Mangatx manga URL" instead?

**Review by hkalexling — CHANGES_REQUESTED** [review]

File: plugins/mangatx/index.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
+var headers = {
+	'Cookie': 'nw=1',
```
Comment (hkalexling) [inline plugins/mangatx/index.js]: I believe you copied this from the E-Hentai plugin. `nw` means "never warn me again on those potentially offensive galleries". I guess it is not necessary for mangatx?

File: plugins/mangatx/info.json (hunk 1 above)
Comment (robzimpulse) [reply plugins/mangatx/info.json]: sure, updated on commit 73ce376

File: plugins/mangatx/index.js (hunk 2 above)
Comment (robzimpulse) [reply plugins/mangatx/index.js]: yes, I forgot to clean that part. this changes included on commit 73ce376

**Review by hkalexling — APPROVED** [review]

**Comment by hkalexling** [comment]
Thanks!

**Pull request merged**